generated/proto/persist_fs_commitlog.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+package schema;
+
+message CommitLogInfo {
+	int64 start = 1;
+	int64 duration = 2;
+	int64 index = 3;
```

Comment:
Is this the index in the series of commit logs for one block period? Should we put it in the filename instead so we know which file to read first?